web/Views/Property/PropertyInputForm/TenantPreferences.cshtml

```diff
@@ -45,7 +45,7 @@
     <div class="form-check">
         <input type="checkbox" asp-for="AcceptsBenefits" class="form-check-input"/>
         <label asp-for="AcceptsBenefits" class="form-check-label">
-            on Universal Credit or housing benefits?
+            on Housing Benefits?

```

Comment:
What's the reason for this change?

Reply:
The line above says "on universal credit" and affects acceptsCredit, it makes no sense to have the on universal credit bit here any more